package.json

```diff
@@ -1,9 +1,9 @@
 {
     "name": "endava-praksa",
     "version": "1.0.0",
-    "description": "cypress, QA, automation",
+    "description": "cypress, Axios, QA, automation",
     "scripts": {
-        "test": "cypress run"
+        "test": "cucumber-js e2e/features/ -r e2e/step_definitions/ --format json:report/cucumber_report.json"
```

Comment:
this will not work - now you don't have report folder nor cucumber_report.json file
change this so that cucumber_report.json is created on run
you can have report folder on git - but empty